We have no uv support.  Plain tox usage is enough to run tests.

--- click-8.4.1/pyproject.toml.orig
+++ click-8.4.1/pyproject.toml
@@ -164,7 +164,6 @@
 
 [tool.tox.env_run_base]
 description = "pytest on latest dependency versions"
-runner = "uv-venv-lock-runner"
 package = "wheel"
 wheel_build_env = ".pkg"
 constrain_package_deps = true
